app/Http/Kernel.php

```diff
@@ -35,6 +35,7 @@ class Kernel extends HttpKernel
             \Illuminate\View\Middleware\ShareErrorsFromSession::class,
             \App\Http\Middleware\VerifyCsrfToken::class,
             \Illuminate\Routing\Middleware\SubstituteBindings::class,
+            \Laravel\Passport\Http\Middleware\CreateFreshApiToken::class,
```

Comment:
Can you explain the role of this please?

Reply:
This comes from Laravel documents, [Consuming Your API With JavaScript](https://laravel.com/docs/5.7/passport#consuming-your-api-with-javascript). Before I added this, I can't consume the API `/api/devices`.

Reply:
In fact, I'm a little bit of curious how you consume these APIs before?